platform/accessibility/guidance/post-launch-audit-processes.md

```diff
@@ -0,0 +1,19 @@
+# Accessibility Post-Launch Audit Processes
+Post-launch audit processes are designed to evaluate applications and static content using a wider set of devices than the [accessibility staging review](./staging-review-processes.md). The accessibility specialist(s) will also schedule your application or content for a VA 508 office quick test.
+
+You can [request a full audit](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=1Copenut%2C+jenstrickland&labels=508%2FAccessibility%2C+product+support&template=full-accessibility-and-508-office-audit.md&title=Full+Accessibility+%26+508+Office+Audit+%5BFeature-Name%5D) when your application is relatively code-stable in production. Ideally all issues identified during the staging review have been fixed, but your accessibility specialist will communicate known issues to the VA 508 office.
```

Comment:
I worry teams won't remember to request a full audit, as they're on to the next project. Might we start the full audit after the VA 508 Office Review, and after the team says issues from the staging & 508 office reviews are addressed?

Reply:
I think waiting for the VA 508 office to finish their quick test before we start an audit is going to create a longer tail than we want. It's better to run then in parallel whenever possible. Resources and time are harder to come by the farther we get from launch day.

Reply:
I'm still worried teams will forget to request a full audit. Can we have them include us on notice of launch, and then we initiate the full audit?  I may be needlessly worried. If we find we aren't getting notified, we can revisit.